### PR TITLE
Add support for Cantabular v9.2.0

### PR DESCRIPTION
**Pull request by phynes-sensiblecode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,45 +34,56 @@ Basic logging will be displayed by default, including the number of high-level C
...
-t=2022-04-28 15:21:06,357 lvl=INFO msg=Dropped non public classification: CLASS_PRIV
-t=2022-04-28 15:21:06,357 lvl=INFO msg=Loaded metadata for 5 Cantabular variables
-t=2022-04-28 15:21:06,358 lvl=INFO msg=Loaded metadata for 3 Cantabular datasets
-t=2022-04-28 15:21:06,359 lvl=INFO msg=Written dataset metadata file to: ctb_metadata_files/cantabm_v9-3-0_unknown-metadata-version_dataset-md_20220428-1.json
-t=2022-04-28 15:21:06,360 lvl=INFO msg=Dropped non public ONS Dataset: DS_PRIV
-t=2022-04-28 15:21:06,360 lvl=INFO msg=Loaded metadata for 4 Cantabular tables
-t=2022-04-28 15:21:06,360 lvl=INFO msg=Written table metadata file to: ctb_metadata_files/cantabm_v9-3-0_unknown-metadata-version_tables-md_20220428-1.json
-t=2022-04-28 15:21:06,360 lvl=INFO msg=Loaded service metadata
-t=2022-04-28 15:21:06,360 lvl=INFO msg=Written service metadata file to: ctb_metadata_files/cantabm_v9-3-0_unknown-metadata-version_service-md_20220428-1.json
+t=2022-05-09 21:26:50,348 lvl=INFO msg=ons_csv_to_ctb_json_main.py version 1.1.alpha
```
Comment (SpencerHedger) [inline README.md:37]: Introduction text still states: "It is compatible with version `1.1` of the metadata schema and version `9.3.0` of `cantabular-metadata`."

**Review by SpencerHedger — CHANGES_REQUESTED** [review]
Works well, downloaded 9.2.0 and tried that as well as the newer format and both seem to be loading correctly etc.

Some questions around the allowed version numbers, but depending upon your intent with this, it might not need any changes.

File: README.md (hunk 1 above)
Comment (phynes-sensiblecode) [reply README.md:37]: OK, will update.

**Review by SpencerHedger — APPROVED** [review]
LGTM.

**Pull request merged**